src/systems/time_control.rs

```diff
@@ -0,0 +1,87 @@
+use amethyst::{
+    prelude::*,
+    core::timing::Time,
+    assets::{Directory, AssetStorage, Loader},
+    ecs::*,
+    shrev::{
+        EventChannel, ReaderId,
+    },
+    ui::*,
+};
+
+pub fn create_time_control_ui(world: &mut World) {
+    world.add_resource(AssetStorage::<FontAsset>::new());
+    let font_handle = {
+        let mut loader = world.write_resource::<Loader>();
+        loader.add_source("fonts", Directory::new("D:/Rust/evoli/resources/assets/fonts"));
```

Comment:
Shouldn't this be a relative path?

Reply:
It probably should, pong has an example of an approach for this: https://github.com/amethyst/amethyst/blob/master/examples/pong/main.rs#L44

Reply:
Of course it should. I forgot to change it... 😅

Reply:
Should be fixed now.